tests/rbd/test_rbd_namespace_same_img_diff_ns.py

```diff
@@ -0,0 +1,85 @@
+from ceph.rbd.initial_config import initial_rbd_config
+from ceph.rbd.workflows.cleanup import cleanup
+from ceph.rbd.workflows.namespace import create_namespace_and_verify
+from ceph.rbd.workflows.rbd import create_single_image
+from ceph.utils import get_node_by_id
+from cli.rbd.rbd import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_namespace_creation_default_pool(client, **kw):
+    """
+    Tests img creation with the same name in diff ns
+    Args:
+        client: rbd client obj
+        **kw: test data
+    """
+    kw["client"] = client
+
+    # for all the pools mentioned in the config perform the test
+    for pool, _ in kw["config"]["rep_pool_config"].items():
+        # namespace1 creation in pool
+        kw["namespace"] = "ns1"
+        rc_ns1 = create_namespace_and_verify(**kw)
+        if rc_ns1 != 0:
+            log.error(f"Namespace creation {kw['namespace']} failed")
+            return 1
+        # img creation in namespace ns_default_pool_1 in the default rbd pool
+        image_config_val = {"namespace": kw["namespace"], "size": "1G"}
+        rc_ns1_img1 = create_single_image(
+            {}, "ceph", Rbd(client), pool, None, "ns_image", image_config_val, None
+        )
+        if rc_ns1_img1 != 0:
+            log.error(f"Image creation failed in namespace {kw['namespace']}")
+            return 1
+
+        # namespace2 creation in pool
+        kw["namespace"] = "ns2"
+        rc_ns2 = create_namespace_and_verify(**kw)
+        if rc_ns2 != 0:
+            log.error(f"Namespace creation {kw['namespace']} failed")
+            return 1
+        # img with same name creation in namespace ns_default_pool_1 in the default rbd pool
+        image_config_val = {"namespace": kw["namespace"], "size": "1G"}
+        rc_ns1_img2 = create_single_image(
+            {}, "ceph", Rbd(client), pool, None, "ns_image", image_config_val, None
+        )
+        if rc_ns1_img2 != 0:
+            log.error(f"Image creation failed in namespace {kw['namespace']}")
+            return 1
+    return rc_ns1_img2
+
+
+def run(**kw):
+    """RBD img creation with the same name in diff ns.
```

Comment:
Please provide deatiled info instead of short form otherwise it's difficult for new user to understand

Reply:
I followed https://github.com/red-hat-storage/cephci/blob/master/tests/rbd/test_rbd_image_operations.py as a reference, can add all the steps from polarion here but it will be duplicate